core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1311,6 +1316,16 @@ public static Sampler<? super RequestContext> requestContextLeakDetectionSampler
         return REQUEST_CONTEXT_LEAK_DETECTION_SAMPLER;
     }
 
+    /**
+     * Returns the {@link CompositeMeterRegistry} where armeria records metrics to by default.
+     *
+     * <p>The default value of this flag is {@link Metrics#globalRegistry}.
+     */
+    @UnstableApi
+    public static CompositeMeterRegistry meterRegistry() {
```

Comment:
How about using `MeterRegistry` instead to give users more freedom of choice?

Reply:
I intentionally used `CompositeMeterRegistry` because I imagined the use-case would be like..

```
PrometheusMeterRegistry prometheusMeterRegistry = new PrometheusMeterRegistry(...);
Flags.meterRegistry().add(prometheusMeterRegistry);
```

which we can't do if `Flags.meterRegistry()` is of `MeterRegistry` type.
I imagined this is why `Metrics.globalRegistry` is also of `CompositeMeterRegistry` type.

What do you think of leaving as-is?

Reply:
That also makes sense. 👍